### DISCUSSION TITLE
Super mario

### DISCUSSION THREAD

**Discussion by Test0102tr** [skyline-emu/skyline#1432] [Ideas]
To increase the emulation speed. I recommend that you start rendering Mario's floor like this, but it's easy to return the same color as the floor

**Comment by Test0102tr** [comment]
![IMG_20220531_142731](https://user-images.githubusercontent.com/106497002/171236150-9634eda4-f798-4040-b796-e5ae5cdf25a9.jpg)

**Reply by Test0102tr** [reply]
the same color

**Comment by riperiperi** [comment]
Yes, I agree with this assessment. On my device, the emulator looks like this:

![image](https://user-images.githubusercontent.com/6294155/171258959-d1433835-2668-4b28-8549-040a37f8375a.png)

But I would like it to look more like this:

![fastest mario](https://user-images.githubusercontent.com/6294155/171259438-7691b287-02d2-4ff5-bb54-076684f67077.png)

I've computed this helpful difference map to let you see what the differences are, and how exactly the colours should be changed to better match the image:

![fastest mario difference](https://user-images.githubusercontent.com/6294155/171259528-e8e89368-5fde-4817-bf4c-8ca64d59764e.png)

Please get back to me on this idea. I think it will really help you out.

**Reply by PixelyIon** [reply]
We will be integrating these changes at the earliest, I am hard coding the image into the APK as we speak.

**Reply by Test0102tr** [reply]
Wow  amazing
